homeassistant/components/fido/sensor.py

```diff
@@ -97,32 +182,18 @@ def __init__(self, fido_data, sensor_type, name, number):
         self.client_name = name
         self._number = number
         self.type = sensor_type
-        self._name = SENSOR_TYPES[sensor_type][0]
-        self._unit_of_measurement = SENSOR_TYPES[sensor_type][1]
-        self._icon = SENSOR_TYPES[sensor_type][2]
+        metadata = SENSOR_TYPES[sensor_type]
+        self._attr_name = f"{self.client_name} {self._number} {metadata.key}"
+        self._attr_unit_of_measurement = metadata.unit_of_measurement
+        self._attr_icon = metadata.icon
```

Comment:
When the `self.entity_description` is set, these aren't needed anymore.

```suggestion
```

Reply:
Done.